appimagebuilder/app_dir/deploy/pacman/venv.py

```diff
@@ -160,9 +161,22 @@ def _generate_config(self):
                         f.write("Server = %s\n" % server)
 
     def _configure_keyring(self):
+        keyrings = list(
+            map(
+                lambda x: x.split('.gpg')[0],
+                [os.path.basename(x) for x in glob.glob('/usr/share/pacman/keyrings/*.gpg')]
+            )
+        )
+
+        self._run_command(
```

Comment:
Please make sure that the command is stopped when the appimage-builder process ends.